add new nop command features (#967)
hugsy/gef: docs/commands/nop.md

The `nop` command allows you to easily patch instructions with nops.

```
nop [LOCATION] [--i ITEMS] [--f] [--n] [--b]
```

> `LOCATION` address/symbol to patch (by default this command replace whole instruction(s))

Reviewer: ```suggestion
`LOCATION` address/symbol to patch (by default this command replaces whole instructions)
```

Author: done

`--i ITEMS` number of items to insert (default 1)

> `--f` Force patch when the final instruction/nop can be broken

Reviewer: ```suggestion
`--f` Force patch even when the selected settings could overwrite partial instructions
```

Author: done

> `--n` Instead replace whole instruction(s), insert the number specified by ITEMS-value of nop(s)-instruction(s)

Reviewer: ```suggestion
`--n` Instead of replacing whole instructions, insert ITEMS nop instructions, no matter how many instructions it overwrites
```

Author: done

> `--b` Instead replace whole instruction(s), fill with nop(s)-instruction(s) the number specified by ITEMS-value bytes

Reviewer: ```suggestion
`--b` Instead of replacing whole instructions, fill ITEMS bytes with nops
```

Author: done

> ```bash

Reviewer 2: This command has become a bit convoluted. Can you also add to the example what the commands do in plain english?

Author: me?? in english??? xDD oka, I will try

Author: done

The `nop` command allows you to easily patch instructions with nops.

```
nop [LOCATION] [--i ITEMS] [--f] [--n] [--b]
```

`LOCATION` address/symbol to patch (by default this command replaces whole instructions)

`--i ITEMS` number of items to insert (default 1)

`--f` Force patch even when the selected settings could overwrite partial instructions

> `--n` Instead of replacing whole instructions, insert ITEMS nop instructions, no matter how many instructions it overwrites

Reviewer: wrap to 100 according to editorconfig

Author: done

Reviewer: I don't think it is?

`--b` Instead of replacing whole instructions, fill ITEMS bytes with nops

nop the current instruction ($pc):
```bash
gef➤ 	nop
```

nop an instruction at $pc+3 address:
```bash
gef➤ 	nop $pc+3
```

nop two instructions at address $pc+3:
```bash
gef➤ 	nop --i 2 $pc+3
```

Replace 1 byte with nop at current instruction ($pc):
```bash
gef➤ 	nop --b
```

> Replace 1 byte with nop at $pc+3 address:

Reviewer: ```suggestion
Replace 1 byte with nop at address $pc+3:
```

Author: done

```bash
gef➤ 	nop --b $pc+3
```

> Replace 2 bytes with nop(s) (breaking the last instruction) at $pc+3 address:

Reviewer: ```suggestion
Replace 2 bytes with nop(s) (breaking the last instruction) at address $pc+3:
```

```bash
gef➤ 	nop --f --b --i 2 $pc+3
```

> Patching with 2 nops at $pc+3 address:

Reviewer: Patch 2 nops at address $pc+3
